hubmap/frontend/cypress/integration/hubmap_spec.js

```diff
@@ -29,6 +47,20 @@ describe('HuBMAP', () => {
 
     // Footer:
     cy.contains('Supported by the NIH Common Fund');
+
+    cy.get('#loggedin-menu').should('exist');
+
+    // auth cookie should be present
+    cy.getCookie('email').should('exist');
+
+    // UI should reflect this user being logged in
+    cy.get('#button-menu').click().then((el) => {
+      assert.include(el[0].textContent, 'Logged in');
+      cy.get('#loggedin-menu').find('li').as('menuitems').then((menuitem) => {
```

Comment:
Again, writing tests that depend on element IDs and DOM structure is deprecated: https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements ... though I think for us going all the way to `cy-data` is too probably far: Makes the tests less readable, for a small decrease in fragility. Our site is small enough that if the wording did change, that would be worth changing the test, I think.

Reply:
well I have been telling i am 2 days old in cypress. And also I asked some questions on your review comments. You are not addressing my questions, in order to avoid this kind of situation. It is really good, if you can be very **specific** in your review comments, given that I , perhaps we, have established I am new to cypress ?